go.mod

```diff
@@ -49,3 +54,5 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.0+incompatible
 	k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
 )
+
+replace github.com/dapr/components-contrib => ../components-contrib
```

Comment:
The components side PR https://github.com/dapr/components-contrib/pull/434 needs to go in before this one.  This line won't be in the final version.